Windows/MainWindow.xaml.cs

```diff
@@ -538,21 +642,11 @@ public partial class MainWindow {
             }
         }
 
-        [CanBeNull]
-        private string CreateWildcardRegex(string uniqueId) {
-            if (!uniqueId.Contains('|')) return null;
-            var split = uniqueId.Split('|');
-            for (var i = 0; i < split.Length; i++) {
-                split[i] = $@"({split[i]}|\*)";
-            }
-            return $"^{string.Join(@"\|", split)}$";
-        }
-
         private async void SaveJson(bool mergeWithTarget) {
             if (string.IsNullOrEmpty(targetFile)) return;
 
             try {
-                var         fileName      = Path.GetFileName(targetFile);
+                var fileName = Path.GetFileName(targetFile);
```

Comment:
Please don't make pointless changes to my style like this.

Reply:
Again, I'm sorry if I made this change.  I agree that it is absolutely pointless.